src/resolvers/donationResolver.test.ts

```diff
@@ -1570,3 +1572,96 @@ function donationsByDonorTestCases() {
     );
   });
 }
+
+function donationsToWalletsTestCases() {
+  const walletAddress1 = generateRandomEtheriumAddress();
+  const walletAddress2 = generateRandomEtheriumAddress();
+  const walletAddress3 = generateRandomEtheriumAddress();
+  const walletAddress4 = generateRandomEtheriumAddress();
```

Comment:
It's better to define all needed variables inside the test case, not in  `donationsToWalletsTestCases()` function, you can move it to arrow function of test case, so in there you can just say `walletAddress` and you don't need to add a  number suffix to them